st2common/st2common/util/config_loader.py

```diff
@@ -224,7 +224,7 @@ def _get_datastore_value_for_expression(self, key, value, config_schema_item=Non
 def get_config(pack, user):
     """Returns config for given pack and user.
     """
-    LOG.debug('Attempting to get config')
+    LOG.debug('Attempting to get config for pack "%s" and user "%s"' % (pack, user))
```

Comment:
Originally this log message was not all that useful because it didn't include pack and username so you weren't able to spot ``pack`` is ``None`` just by looking at the log messages.